cpp/src/slice2cs/Gen.cpp

```diff
@@ -267,14 +267,22 @@ Slice::CsVisitor::writeMarshalDataMembers(const DataMemberList& p, const string&
 #ifndef NDEBUG
     int currentTag = -1; // just to verify sortForMarshaling sorts correctly
 #endif
+
+    int bitSequenceIndex = -1;
```

Comment:
Why do we need to initialize it to -1 in this case?

Reply:
It's not necessary, and we could just initialize it to 0. The -1 invalid value could trigger an assert in `writeMarshalCode` if there is a bug ... but there is no bug.